src/browser/webview/mattermost.js

```diff
@@ -194,6 +194,18 @@ function setSpellChecker() {
 setSpellChecker();
 ipcRenderer.on('set-spellchecker', setSpellChecker);
 
+ipcRenderer.on('user-is-active', () => {
+  if (window.desktopBridge && window.desktopBridge.updateUserActivityStatus) {
+    window.desktopBridge.updateUserActivityStatus(true);
+  }
+});
+
+ipcRenderer.on('user-is-inactive', () => {
+  if (window.desktopBridge && window.desktopBridge.updateUserActivityStatus) {
```

Comment:
I know I'm biased here, but 🎉

Reply:
Redid the bridge to mimick the existing functionality of Electron's ipcMain and ipcRenderer modules already used for this feature.